### PR TITLE
Updates Judge Recusal Box Style

### PR DESCRIPTION
**Pull request by devkonrad**
Updates Judge Recusal Box Style

Changes:
modified: app/javascript/judge/scores/JudgeRecusalPopup.vue

Sample of the new box style:
recusal-popup

Refs: https://github.com/Iridescent-CM/technovation-app/issues/3409

### CODE REVIEW

**Comment by dboyer** [comment]
Completed a functionality/style review and confirmed this is ready for code review @viviancan

**Review by viviancan — CHANGES_REQUESTED** [review]
I just left a follow up comment for Deb regarding the styling! I will wait for confirmation before the approval. Thx, Alex 💯 

The other thing I noticed is that the font for the recusal options seems to follow the older styling. Not sure if we want to update that 🤔

File: app/javascript/judge/scores/JudgeRecusalPopup.vue [hunk 1]
```diff
@@ -77,12 +77,13 @@ export default {
...
         confirmButtonText: 'Remove me from this submission',
-        confirmButtonColor: '#a50009',
+        confirmButtonColor: '#3FA428',
         showCancelButton: true,
         cancelButtonText: 'I want to go back and try judging',
         focusConfirm: false,
         reverseButtons: true,
-        width: '70%'
+        width: '45%',
+        customClass: 'tw-green-btn'
```
Comment (viviancan) [inline app/javascript/judge/scores/JudgeRecusalPopup.vue:86]: This custom class of `tw-green-btn` is causing the entire modal to turn green :)

**Comment by devkonrad** [comment]
> I just left a follow up comment for Deb regarding the styling! I will wait for confirmation before the approval. Thx, Alex 💯 
> 
> The other thing I noticed is that the font for the recusal options seems to follow the older styling. Not sure if we want to update that 🤔 

Thanks, @viviancan . Im going to take a look in these points. 😉

File: app/javascript/judge/scores/JudgeRecusalPopup.vue (hunk 1 above)
Comment (devkonrad) [reply app/javascript/judge/scores/JudgeRecusalPopup.vue]: Thanks for the catch, @viviancan ! The bug was fixed! :)

**Review by viviancan — APPROVED** [review]
Did not pull down again! LGTM!

**Comment by devkonrad** [comment]
> Did not pull down again! LGTM!

Thanks for the review, @viviancan! Im going to merge it! :)

**Pull request merged**